ngraph/core/include/ngraph/op/round.hpp

```diff
@@ -54,5 +56,43 @@ namespace ngraph
         NGRAPH_SUPPRESS_DEPRECATED_START
         using v0::Round;
         NGRAPH_SUPPRESS_DEPRECATED_END
+
+        namespace v5
+        {
+            /// \brief Elementwise round operation. The output is round to the nearest integer
+            /// for each value. In case of halfs, the rule is defined in attribute 'mode':
+            ///     'half_to_even' - round halfs to the nearest even integer.
+            ///     'half_away_from_zero': - round in such a way that the result heads away from
+            /// zero.
+
+            class NGRAPH_API Round : public ngraph::op::Op
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+
+                /// \brief Constructs a round operation.
+                Round() = default;
+
+                /// \brief Constructs a round operation.
+                ///
+                /// \param arg Node that produces the input tensor.
+                /// \param mode Rule to resolve halfs
+                Round(const Output<Node>& arg, const std::string& mode);
+
+                bool visit_attributes(AttributeVisitor& visitor) override;
+                void validate_and_infer_types() override;
+
+                virtual std::shared_ptr<Node>
+                    clone_with_new_inputs(const OutputVector& new_args) const override;
+
+                bool evaluate(const HostTensorVector& outputs,
+                              const HostTensorVector& inputs) const override;
+
+                std::string get_mode() const { return m_mode; }
+                void set_mode(const std::string& mode) { m_mode = mode; }
```

Comment:
We use `enum` instead of `string`. Refer to TopK or NMS op for the example. Also, do not forget to add an attribute adapter and update IE IR Parser `inference-engine/src/readers/ir_reader/ie_ir_parser.hpp`

Reply:
Done, now `mode` attribute has `enum` type. Adapter added, I need to check and update parser.

Reply:
Parser added.